streamer/packager_node.py

```diff
@@ -112,7 +112,7 @@ def start(self) -> None:
       # Log by writing all Packager output to a file.  Unlike the logging
       # system in ffmpeg, this will stop any Packager output from getting to
       # the screen.
-      stdout = open('PackagerNode.log', 'w')
+      stdout = open('PackagerNode.log', 'a')
```

Comment:
The log files could have a number added, based on the period, such as `PackagerNode-1.log`.  That would be a nice solution to the problem of overwriting them, I think.